web/frontend/src/components/movement-sensor/index.svelte

```diff
@@ -250,6 +299,33 @@ useConnect(() => {
               {compassHeading?.toFixed(2)}
             </td>
           </tr>
+          {#if accuracy?.compassDegreesError}
+            <tr>
+              <th class="border border-medium p-2"> Compass Degrees Error </th>
+              <td class="border border-medium p-2">
+                {accuracy.compassDegreesError.toFixed(2)}
+              </td>
+            </tr>
+          {/if}
+        </table>
+      </div>
+    {/if}
+
+    {#if accuracy?.accuracyMap}
```

Comment:
nit: don't display the header if there are no values

```suggestion
    {#if (accuracy?.accuracyMap?.size ?? 0) > 0}
```